### DISCUSSION TITLE
Use llama 3 instead of codellama

### DISCUSSION THREAD

**Discussion by magneticmojo** [yusufcanb/tlm#26] [General]
Could you make it configurable to chose llama 3 instead of codellama?

**Comment by yusufcanb** [comment]
There is already a ticket regarding allowing other models. I will investigate and try to understand it's feasibility on computers with limited resources.

**Comment by Trapper4888** [comment]
There is also codeqwen and deepseek 7b that should be good.

I think it would be better to have a default model, let the user optionally change it, and hardcode the custom parameter into the request instead of making 2 modelfils and 2 new alias on ollama side for the request and suggest.
